src/components/SetUp/ThirdForm.jsx

```diff
@@ -0,0 +1,78 @@
+import { Box, VStack, Text, HStack, Flex } from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+
+const ThirdForm = () => {
+  return (
+    <Flex height="90vh" justifyContent="center" alignItems="center" bg="#FFFFFF">
+      <VStack spacing={8}>
+        <Text color={'#359797'} fontSize="5xl" fontWeight="bold">
+          Let's Get You Set Up!
```

Comment:
🚫 **[eslint]** <[react/no-unescaped-entities](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/no-unescaped-entities.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.